docs/bnb-smart-chain/developers/paymaster/paymaster-api.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Paymaster-API - BSC Paymaster
+---
+
+# Paymaster API Spec
+
+To facilitate widespread adoption and ensure interoperability across diverse wallet implementations, it is crucial to establish a standardized set of interface specifications for paymasters. This standardization will enable wallet developers to integrate gas sponsorship features efficiently and consistently, regardless of the specific paymaster service they choose to utilize.
+
+## API Spec
+
+Paymaster needs to implement a JSON-RPC API called `pm_isSponsorable`, so that it can return sponsor and policy information to wallets. Paymaster also needs to implement `eth_sendRawTransaction` JSON-RPC API. The detailed API Specs are defined as below:
+
+### pm\_isSponsorable
+
+**Request Parameters**
+
+*   `jsonrpc`: The JSON-RPC protocol version ("2.0").
+*   `id`: A unique identifier for the request (1 in this example).
+*   `method`: The method name to be invoked ("pm\_isSponsorable").
+*   `params`: An array containing a single object with the following fields:
+    *   `to`: The recipient address of the transaction.
+    *   `from`: The sender address of the transaction.
+    *   `value`: The value of the transaction in hexadecimal.
+    *   `data`: Additional data for the transaction in hexadecimal.
+    *   `gas`: The gas limit of the transaction in hexadecimal.
+
+**Example:**
+
+```plain
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "pm_isSponsorable",
+  "params": [
+    {
+      "to": "0x...", // an address
+      "from": "0x...", // an address"value": "0xa1",
+      "data": "0x",
```

Comment:
shall we add "value" param in the example?

Reply:
fixed.